src/Tests/Microsoft.Diagnostics.Monitoring.UnitTestApp/Microsoft.Diagnostics.Monitoring.UnitTestApp.csproj

```diff
@@ -13,11 +13,12 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\Microsoft.Diagnostics.Monitoring.HostingStartup\Microsoft.Diagnostics.Monitoring.HostingStartup.csproj" />
+    <ProjectReference Include="..\..\Microsoft.Diagnostics.Monitoring.StartupHook\Microsoft.Diagnostics.Monitoring.StartupHook.csproj" />
```

Comment:
Why is this being included in the unit test app? This means that the assembly is in the application's default probing path, which we want to avoid since we want to mimic real applications as much as possible.

Reply:
This project also tests Function Probes which are now in the StartupHook assembly. Should I reference the source files in the unit test project so I can remove the assembly dependency, or should I create a separate unit tests project for function probes scenario?

Reply:
Chatted with @clguiman about this. This change isn't introducing a new dependency on the functionality that is moving. Let's keep it as you have it and then we can separately consider if a new app should be created for testing these.